src/serve_rendered.js

```diff
@@ -1005,6 +1005,7 @@ export const serve_rendered = {
         );
       }
       const info = clone(item.tileJSON);
+      info.tileSize = (tileSize != undefined ? tileSize : 256)
```

Comment:
*cf.* Line 997 where `tileSize` is set

```javascript
const tileSize = parseInt(req.params.tileSize, 10) || undefined
```

Reply:
ya, I made this default to undefined for a reason. When I added 512 tiles and added tileSize, I wanted to preserve the old behavior when the user when to the old /style/id.json endpoint urls (which returned a 256px tile before). When it is undefined, getTileUrls returns a tile url that does not have a tilesize included in the tiles url (like it did before). When tileSize is defined, the size gets included in the tiles url.